src/features/MoviesList/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import MovieTile from "../../common/MovieTile";
+import { MainContainer, StyledMainHeader } from "./styled";
+import { useMoviesData } from "./useMoviesData";
+
+export const MoviesList = () => {
+    const moviesData = useMoviesData();
+
+    return (
+        <> <MainContainer>
+            <StyledMainHeader>Popular movies</StyledMainHeader>
+
+            {moviesData.status === "loading" ? (
+                <StyledMainHeader>Please WAIT</StyledMainHeader>
+            ) : moviesData.status === "error" ? (
+                <StyledMainHeader>SOMETHING WENT WRONG, sorry.. </StyledMainHeader>
+            ) : (
+                moviesData.results.map(element => (
+                    <React.Fragment key={element.id}>
+                        <MovieTile
+                            posterPath={element.poster_path}
+                            title={element.title}
+                            year={new Date(element.release_date).getFullYear()}
+                            rating={element.vote_average}
+                            votes={element.vote_count}
+                            tag={element.genre_ids}
+                        />
+                    </React.Fragment>
+                )))}
+
+        </MainContainer>
+        </>
+    )
+
+};
```

Comment:
Code doesn't seem to be formatted here. I highly recommend installing prettier extension for VSC and setting VSC formatting on file save 💪

Reply:
interesting - I already got that set :D 
But what should be different ? :)